packages/data-point/lib/reducer-types/reducer-list/factory.test.js

```diff
@@ -105,4 +105,28 @@ describe('factory#create', () => {
     expect(result.reducers[2].type).toBe('ReducerPath')
     expect(result.reducers[3].type).toBe('ReducerFunction')
   })
+
+  test('factory#throw error if reducer is false', () => {
+    expect(() => factory.create(createReducer, [false])).toThrow()
+  })
+
+  test('factory#throw error if reducer is empty string', () => {
+    expect(() => factory.create(createReducer, '')).toThrow()
+  })
+
+  test('factory#throw error if reducer is non-empty string with no matching reducer', () => {
+    expect(() => factory.create(createReducer, 'asdf')).toThrow()
+  })
+
+  test('factory#throw error if one of two reducers is false', () => {
+    expect(() => factory.create(createReducer, ['$foo.bar', false])).toThrow()
+  })
+
+  test('factory#throw error if reducer is undefined', () => {
+    expect(() => factory.create(createReducer, [undefined])).toThrow()
+  })
+
+  test('factory#throw error if reducer is zero', () => {
```

Comment:
"factory#throw" can be replaced with "should throw" - the `#` is normally used to indicate a method/property, like how `factory#create` is used because `create` is a method of `factory`